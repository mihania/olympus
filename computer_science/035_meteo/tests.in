3
3 -14.23 15.12
Ianuarie
2
4.6
-23.6
Februarie
2
-16
6
Martie
3
13.56
16.45
13
5 4.23 25.12
Ianuarie
2
4.6
-23.6
Martie
3
13.56
16.45
6
Iunie
2
23.56
13
Iulie
2
26.56
16
Decembrie
2
-6.56
8
4 -30 40
Februarie
3
-16.79
-15.68
-29.45
Aprilie
3
7.39
-9.14
18.98
Iunie
4
33.81
30.77
25.17
20.13
Decembrie
2
-29.53
-19.65

