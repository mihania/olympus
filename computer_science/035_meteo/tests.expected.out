NU
Iunie
Martie
Aprilie
Decembrie
Februarie
Iunie
